data/sound-strings/pl.json/localize.json

```diff
@@ -3,13 +3,13 @@
 "make_a_right_turn":"Skręć w prawo.",
 "make_a_sharp_right_turn":"Ostro w prawo.",
 "enter_the_roundabout":"Wjedź na rondo.",
-"leave_the_roundabout":"Zjazd z ronda.",
+"leave_the_roundabout":"Zjedź z ronda.",
 "make_a_slight_left_turn":"Trzymaj się lewej strony.",
 "make_a_left_turn":"Skręć w lewo.",
 "make_a_sharp_left_turn":"Ostro w lewo.",
 "make_a_u_turn":"Zawróć.",
 "go_straight":"Jedź prosto.",
-"exit":"Zjazd.",
+"exit":"Zjedź w prawo",
 "destination":"Dojedziesz.",
```

Comment:
I think the better version would be "Dotrzesz do celu." or "Dojedziesz do celu.". @lunarna-gh

Reply:
I wasn't sure of this, because of the example string "In five hundred feet You’ll arrive onto Exit 12, Main Street, London Then Take the first exit." -- would it sound good if it was used for midway points as well?

Reply:
@lunarna-gh It's only used at the absolute end of the route. It's not used in middle points. It's for example used like "Za 100 metrów jedź prosto, następnie dojedziesz". I think that @zyphlar probably auto-generated that and it made some incorrect artifacts. Look at the english version https://github.com/organicmaps/organicmaps/blob/master/data/sound-strings/en.json/localize.json#L13